infra/resources/_modules/key_vaults/io_com_key_vault.tf

```diff
@@ -0,0 +1,22 @@
+resource "azurerm_key_vault" "com" {
+  name = provider::dx::resource_name({
+    prefix          = "io"
+    name            = "com",
+    domain          = "",
+    resource_type   = "key_vault",
+    environment     = "p",
+    location        = "italynorth"
+    instance_number = 1
```

Comment:
please replace these hardcoded values with variables (use a variable named `environment` like the one used in `web_apps`)

`merge(var.environment, { resource_type: "key_vault", instance_number: 1 })`

Reply:
[ea5f195](https://github.com/pagopa/io-messages/pull/260/commits/ea5f19584e4151ea4937ba0636ce74042a1a1a34)